src/MagnumPlugins/WebPImporter/Test/WebPImporterTest.cpp

```diff
@@ -0,0 +1,411 @@
+            }), TestSuite::Compare::Container);
+            break;
+
+        /* Same image with 90% image quality (lossy bitstream) */
+        case 1:
+            CORRADE_COMPARE_AS(image->data(), Containers::arrayView<char>({
+                '\x60', '\x4a', '\x2b', '\x80',
+                '\x60', '\x4a', '\x2b', '\xff',
+                '\x60', '\x4a', '\x2b', '\x80',
+
+                '\xbb', '\xa9', '\xad', '\xff',
+                '\xbb', '\xa9', '\xad', '\xff',
+                '\xbb', '\xa9', '\xad', '\xff',
+
+                '\x5d', '\x53', '\x9d', '\x80',
+                '\x5d', '\x53', '\x9d', '\xff',
+                '\x5d', '\x53', '\x9d', '\x80',
+            }), TestSuite::Compare::Container);
+            break;
+
+        /* Same image with 45% image quality (lossy bitstream) */
+        case 2:
+            CORRADE_COMPARE_AS(image->data(), Containers::arrayView<char>({
+                '\x5d', '\x4b', '\x2f', '\x80',
+                '\x5d', '\x4b', '\x2f', '\xff',
+                '\x5d', '\x4b', '\x2f', '\x80',
+
+                '\xb7', '\xa8', '\xb0', '\xff',
+                '\xb7', '\xa8', '\xb0', '\xff',
+                '\xb7', '\xa8', '\xb0', '\xff',
+
+                '\x5d', '\x58', '\xa2', '\x80',
+                '\x5d', '\x58', '\xa2', '\xff',
+                '\x5d', '\x58', '\xa2', '\x80',
+            }), TestSuite::Compare::Container);
+            break;
+
+        /* Same image with 0% image quality (lossy bitstream) */
+        case 3:
+            CORRADE_COMPARE_AS(image->data(), Containers::arrayView<char>({
+                '\x2c', '\x31', '\x12', '\x80',
+                '\x2c', '\x31', '\x12', '\xff',
+                '\x2c', '\x31', '\x12', '\x80',
+
+                '\xcb', '\xcd', '\xbe', '\xff',
+                '\xcb', '\xcd', '\xbe', '\xff',
+                '\xcb', '\xcd', '\xbe', '\xff',
+
+                '\x5e', '\x5c', '\x6c', '\x80',
+                '\x5e', '\x5c', '\x6c', '\xff',
+                '\x5e', '\x5c', '\x6c', '\x80',
+            }), TestSuite::Compare::Container);
+            break;
+    }
+}
+
+void WebPImporterTest::openMemory() {
+    auto&& data = OpenMemoryData[testCaseInstanceId()];
+    setTestCaseDescription(data.name);
+
+    Containers::Pointer<AbstractImporter> importer = _manager.instantiate("WebPImporter");
+    Containers::Optional<Containers::Array<char>> memory = Utility::Path::read(Utility::Path::join(WEBPIMPORTER_TEST_DIR, "rgb-lossless.webp"));
+    CORRADE_VERIFY(memory);
+    CORRADE_VERIFY(data.open(*importer, *memory));
+
+    Containers::Optional<Trade::ImageData2D> image = importer->image2D(0);
+    CORRADE_VERIFY(image);
+    CORRADE_COMPARE(image->size(), Vector2i(3, 3));
+    CORRADE_COMPARE(image->format(), PixelFormat::RGB8Unorm);
+
+    CORRADE_COMPARE_AS(image->pixels<UnsignedByte>().asContiguous(), Containers::arrayView<UnsignedByte>({
+        1, 2, 3,
+        4, 5, 6,
+        7, 8, 9
+    }), TestSuite::Compare::Container);
```

Comment:
Ah well ... in case of PngImporter it's because it uses a greyscale file (so each pixel is 1 byte), and then the image has no row padding, so there the test can use `asContiguous()`. Here you have 3-byte RGB pixels and row padding, so you can't.

It would be probably easiest to just copy the comparison of (just lossless) `data()` from the `rgb()` test, it can't get much shorter than that anyway.